README.md

```diff
@@ -95,6 +95,12 @@ See the [Benchmarks](docs/benchmarks.md) document for comparisons.
 <a name="team"></a>
 ## The Team
 
+### Bundling support
+
+Pino supports to being bundled using tools like webpack or esbuild. 
+
+See [Bundling](docs/bundling.md) document for more informations.
```

Comment:
Why is this under the `## The Team` heading?

Reply:
Because I misplaced it. :D 
Will fix soon, thanks for noticing!

Reply:
Fixed!